packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -132,6 +150,43 @@ export const IndexesToolbar: React.FunctionComponent<IndexesToolbarProps> = ({
                 signals={PerformanceSignals.get('too-many-indexes')}
               />
             )}
+            {isSearchManagementActive && (
+              <SegmentedControl
+                onChange={onChangeIndexesSegment}
+                className={alignSelfEnd}
```

Comment:
I don't know what our conventions state, but this might as well be an inline style={{}}?

Reply:
(nitpicky, I know)

Reply:
I saw several places where we use inline styles, I'm fine with either solution. I used this one because that specific file didn't have inline styles and used css-in-js.